source/_integrations/cover.mqtt.markdown

```diff
@@ -421,7 +421,67 @@ cover:
 ```
 
 {% endraw %}
+### Full configuration using advanced templating
 
+The example below shows a full example of how to setup a venetian blind which has a combined position and tilt topic. The blind in the example has moveable slats which tilt with a position change. In the example it takes the blind 6% of the movement for a full rotation of the slats.
+
+Following variable might be used in `position_template`, `set_position_template`, `tilt_command_template` and `tilt_status_template`, `json_attributes_template` (only `entity_id`).
+
+ * `entity_id` - The ID of the entity itself. It can be used to reference its attributes with the help of the [states](/docs/configuration/templating/#states) template function.
+ * `position_open`
+ * `position_closed`
+ * `tilt_min`
+ * `tilt_max`
```

Comment:
```suggestion
- `entity_id` - The ID of the entity itself. It can be used to reference its attributes with the help of the [states](/docs/configuration/templating/#states) template function.
- `position_open`
- `position_closed`
- `tilt_min`
- `tilt_max`
```

Additionally, it is unclear to me what this list is? It is not described.